fullscreen.bs

```diff
@@ -365,7 +362,9 @@ attribute's getter must run these steps:
    <a lt="unfullscreen a document">unfullscreen <var>descendantDoc</var></a>.
    <!-- cross-process -->
 
-   <li><p>For each <var>exitDoc</var> in <var>exitDocs</var>, in order,
+   <li><p>For each <var>exitDoc</var> in <var>exitDocs</var>, in order: If <var>exitDoc</var> is
```

Comment:
Not sure, whatever matches the code. But I would use a sublist after "in order" to deal with the if/otherwise situation in two list items rather than jamming it all together.